src/Amadeus.Server/appsettings.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
This appsettings.json should be kept on par with the development version. (this one is the settings that will be available on every configurations so a connection string should be present.